build/compile/Microsoft.DotNet.Cli.Archive.targets

```diff
@@ -0,0 +1,49 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="GenerateNuGetPackagesArchive"
+            DependsOnTargets="RestoreNuGetPackagesArchive;
+                              CompressNuGetPackagesArchive"
+            Inputs="$(Stage2Directory)/sdk/**/dotnet.dll"
+            Outputs="$(FinalArchive)">
+      <Message Text="Generating NuGet Packages Archive" />
```

Comment:
This message is going to print out after the archive is generated. Seems not quite right given it says `Generating`